model/service_broker/service_plan.go

```diff
@@ -16,7 +16,8 @@ limitations under the License.
 
 package model
 
-//
+// ServicePlan defines on QoS option for a service that a client can
```

Comment:
Can you clarify this comment? Not really sure what it means exactly, specifically QoS